JSDemos/Demos/DataGrid/FilterPanel/description.md

```diff
@@ -1 +1,7 @@
-This example shows how to&nbsp;filter data using the filter panel. You can use its check box to&nbsp;enable/disable the current filter expression, and clicking on&nbsp;this expression opens the integrated filter builder. Note that changes made in&nbsp;it&nbsp;are reflected in&nbsp;the filter row and header filter, and vice versa.
\ No newline at end of file
+[Filter panel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/) is a UI element that displays a combined filter. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterValue) option and consists of filters that users apply in other UI elemens ([filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder)).
+
+To display the filter panel, set the **filterPanel**.[visible](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/#visible) option to **true**. 
+
+A click on the combined filter calls the integrated filter builder. You can configure it in the [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder) object. In this demo, this object is used to add a [custom filter operation](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations).
```

Comment:
Why don't you mention what the custom operation is? You can be specific, so that it's easier for developers to find the code and try the functionality.

Reply:
Sorry, I don;t understand what you mean here. Should I add the value of the custom operation's `name` field here? The fact is we have only one item in the `customOperations` array.
Or did I get you wrong, and I should describe the basics of custom filter operations? Users can do almost anything when they implement custom operations.

Reply:
I meant "for developers to find the code and *for users* to try the functionality". Sorry it came out confusing. All I mean is you can mention what the operation is - what's it called, or what it does. Right now it sounds like "we added a new operation, look for it yourselves" :-)